src/table/src/requests.rs

```diff
@@ -129,6 +134,16 @@ impl TableOptions {
             options.ttl = Some(ttl_value);
         }
 
+        if let Some(skip_wal) = kvs.get(SKIP_WAL_KEY) {
+            options.skip_wal = skip_wal.parse().unwrap_or_else(|_| {
+                warn!(
```

Comment:
Better to return some error to user, otherwise it might cause surprise.

Reply:
fixed in https://github.com/GreptimeTeam/greptimedb/pull/5740/commits/2c067d85a27a9076a7224e15c1e747e2aa957d0f#diff-0d62a3ecad039e20e5695ee9c5e911179c7da14ec7e963f75394e23b7b34f080R137